java/flight/flight-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/client/ArrowFlightSqlClientHandler.java

```diff
@@ -525,12 +543,18 @@ public ArrowFlightSqlClientHandler build() throws SQLException {
 
         if (disableCertificateVerification) {
           clientBuilder.verifyServer(false);
+        } else {
+          if (keyStorePath != null) {
```

Comment:
KeyStorePath should not be used. TrustStorePath should be used instead.

Reply:
Yes, I  forgot to change the naming in here

Reply:
fixed